lib/discourse_chat/provider/telegram/telegram_provider.rb

```diff
@@ -72,6 +72,8 @@ def self.message_text(post)
         I18n.t(
             "chat_integration.provider.telegram.message",
             user: display_name,
+            username: post.user.username,
+            fullname: full_name,
```

Comment:
@eviltrout, the added `username` and `fullname` are not working as expected and when used, give an error: '_The following interpolation key(s) are invalid: "username, fullname"_'. What is the correct way to fix this?
Would adding
```
username = "#{post.user.username}"
fullname = "#{full_name}"
```
before I18n.t() fix the issue? I am new to this and need advice. My thought is they need to be stringified before being passed as `username` and `fullname`.

Reply:
You added them to the call but did not add them to the locale file. Maybe we should revert this?

Reply:
If adding the code above fixes the issue, then I can create a new pull request with them. Currently this PR should not cause any problems. What is your suggestion?